src/apps/tools/OswAppDistStats.cpp

```diff
@@ -15,9 +15,10 @@ void OswAppDistStats::drawChart() {
     uint8_t interval = 20;
     uint16_t goalValue = OswConfigAllKeys::distPerDay.get();
 
-    uint32_t weekDay = 0;
-    uint32_t dayOfMonth = 0;
-    hal->getLocalDate(&dayOfMonth, &weekDay);
+    OSW_DATE oswDate = { 0, };
+    hal->getLocalDate(&oswDate);
+    uint32_t weekDay = oswDate.day;
+    uint32_t dayOfMonth = oswDate.weekDay;
```

Comment:
You are making a copy here... Why? Maybe take the reference `&` or just use the value directly.